### PR TITLE
QA : Tooling (Test Scenarios)

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description  | QA : Tooling (Test Scenarios)
| Fixed ticket | N/A

### CODE REVIEW

**Review by boubkerbribri — APPROVED** [review]

File: content/project-organization/quality-council/tooling/test-scenarios.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+---
+
+# Test Scenarios
+
+## Goal
+
+The Test Scenarios repository focuses on scenarios written by Software Testers (on a JIRA, not accessible to the public). Several tools have been developed:
+
+* a tool for generating Markdown pages from XRay scenarios on JIRA
+* a tool for synchronising scenarios to be automated or in the process of being automated in issues
```
Comment (nicosomb) [inline content/project-organization/quality-council/tooling/test-scenarios.md:12]:
```suggestion
* a tool for synchronizing scenarios to be automated or in the process of being automated in issues
```

File: content/project-organization/quality-council/tooling/test-scenarios.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+* a tool for updating a kanban based on the status of scenarios
+
+## Schedule
+
+* Every week day at 06:00 AM (for the generation of Markdown pages)
+* Every week day at 07:00 AM (for the synchronisation of scenarios in issues)
```
Comment (nicosomb) [inline content/project-organization/quality-council/tooling/test-scenarios.md:18]:
```suggestion
* Every week day at 07:00 AM (for the synchronization of scenarios in issues)
```

File: content/project-organization/quality-council/tooling/test-scenarios.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+* Fetch all XRay scenarios (To Be Automated / Automation In Progress / Automated)
+* For each scenario:
+  * IF the XRay scenario has not linked Github issue
+    * THEN the script create a new issue;
+  * IF the XRay scenario has been updated (title or description)
+    * THEN the script update the Github issue (title & description);
+  * IF the linked Github issue has not the good assignee from JIRA,
+    * THEN the script update the Github issue (assignee);
+  * IF the linked Github issue has no labels, or the linked Github issue has late labels,
```
Comment (nicosomb) [inline content/project-organization/quality-council/tooling/test-scenarios.md:53]:
```suggestion
  * IF the linked Github issue has no label, or the linked Github issue has late labels,
```

**Review by nicosomb — CHANGES_REQUESTED** [review]

File: content/project-organization/quality-council/tooling/test-scenarios.md (hunk 3 above)
Comment (Progi1984) [reply content/project-organization/quality-council/tooling/test-scenarios.md]: Fixed

**Review by nicosomb — APPROVED** [review]

File: content/project-organization/quality-council/tooling/test-scenarios.md (hunk 2 above)
Comment (Progi1984) [reply content/project-organization/quality-council/tooling/test-scenarios.md]: Fixed

File: content/project-organization/quality-council/tooling/test-scenarios.md (hunk 1 above)
Comment (Progi1984) [reply content/project-organization/quality-council/tooling/test-scenarios.md]: Fixed

**Pull request merged**